packages/nocodb/src/lib/noco/meta/api/dataApis/dataAliasApis.ts

```diff
@@ -130,6 +151,10 @@ router.get(
   '/data/:orgs/:projectName/:tableName/views/:viewName',
   ncMetaAclMw(dataList)
 );
+router.get(
+  '/data/:orgs/:projectName/:tableName/views/:viewName/count',
+  ncMetaAclMw(dataCount)
```

Comment:
If you pull the latest code, you will see the change. We add the function name manually instead of reading `.name` property. This is because when we build the backend, the function name will be minified which will cause missing permission in ACL module.

Hence, you can simply change `ncMetaAclMw(dataCount)` to `ncMetaAclMw(dataCount, 'dataCount')`.